CONTRIBUTING.md

```diff
@@ -17,9 +17,21 @@ Please consider these following guidelines when contributing:
 * For any issue regarding memory leaks / high cpu usage please send a [diagnosis](https://nodejs.org/en/docs/guides/simple-profiling/) of your performance 
 * For any uncertainty of how to reproduce the issue, specify that as well! Describe your actions before the bug occured.
 
+### Submitting Enhancements
+We appreciate new ideas and improvements and we want Nino to reach its full potential! 
+If you have any enhancement you want done in Nino please submit an issue!
```

Comment:
Stop shouting! We only need an exclamation mark once here

Reply:
Thanks!